CHANGELOG.md

```
@@ -7,6 +7,17 @@ and Yorkie adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 ## [Unreleased]
 
+## [v0.6.37] - 2025-11-11
+
+### Added
+
+- Introduce hierarchical channel keys by @emplam27 in https://github.com/yorkie-team/yorkie/pull/1559
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)